pkg/operators/limiter/limiter_test.go

```diff
@@ -0,0 +1,78 @@
+// Copyright 2024 The Inspektor Gadget authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package limiter
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestParseMaxEntries(t *testing.T) {
```

Comment:
Ah, those tests might be nice for the [helpers](https://github.com/inspektor-gadget/inspektor-gadget/blob/main/pkg/gadget-service/api-helpers/apihelpers.go#L111) I mentioned 😄